6-downloading-and-running-existing-tasks/README.md

```diff
@@ -14,148 +14,147 @@ In this exercise you will explore some existing tasks, including several tasks t
-minifact
-mysql::sql                    Allows you to execute arbitary SQL
-package                       Manage and inspect the state of packages
-puppet_conf                   Inspect puppet agent configuration settings
-puppet_device                 Run puppet device on a (proxy) Puppet agent
-puppeteer::apply              Run a puppet apply on agents
-puppeteer::certificate_info   Grab certificate information on agent
-puppeteer::external_fact      Add or update external facts on agents
-puppeteer::features           Query the node for what features are available
-puppeteer::providers          Query the node for what providers are available
-resource                      Inspect the value of resources
-service                       Manage and inspect the state of services
-
-Use `bolt task show <task-name>` to view details and parameters for a specific task.
-```
-
-# Use the package task to check status of package
-
-With Puppet installed on the node we can use some of the tasks that expose Puppet resources, like the package task which you just installed from the Puppet Forge with r10k.  We can use bolt to show us the parameters used by the package task.
-
-```
-$ bolt task show package
-package - Manage and inspect the state of packages
-
-USAGE:
-bolt task run --nodes, -n <node-name> package action=<value> name=<value> [version=<value>] [provider=<value>]
-
-PARAMETERS:
-- action: Enum['install', 'status', 'uninstall', 'upgrade']
-    The operation (install, status, uninstall and upgrade) to perform on the package
-- name: String[1, default]
-    The name of the package to be manipulated
-- version: Optional[String[1, default]]
-    Version numbers must match the full version to install, including release if the provider uses a release moniker.
-    Ranges or semver patterns are not accepted except for the gem package provider.
-    For example, to install the bash package from the rpm bash-4.1.2-29.el6.x86_64.rpm, use the string '4.1.2-29.el6'.
-- provider: Optional[String[1, default]]
-    The provider to use to manage or inspect the package, defaults to the system package manager
-```
-
-Let's quickly check on the status of a specific package using `bolt`:
-
-```
-$ bolt task run package action=status name=bash --nodes all
-Started on node1...
-Finished on node1:
-  {
-    "status": "up to date",
-    "version": "4.2.46-29.el7_4"
-  }
-Ran on 1 node in 3.81 seconds
-```
-
-# Use the package task to install a package
-
-The package task also supports other actions, including ensuring a package is installed. Let's install a package across all of our nodes using that action:
-
-```
-$ bolt task run package action=install name=vim --nodes all --run-as root
-Started on node1...
-Finished on node1:
-  {
-    "status": "present",
-    "version": "2:7.4.160-2.el7"
-  }
-Ran on 1 node in 15.26 seconds
-```
+- Run the 'bolt task show' command to view a list of the tasks installed in the previous exersize.
```

Comment:
exersize

Reply:
Fixed here and ran spellcheck just in case. (Evidently I really do not like exercise.)